### DISCUSSION TITLE
Is this "custom firmware" can handle 3-phase/3-conductor systems (Delta systems without neutral conductor)?

### DISCUSSION THREAD

**Discussion by lg2dc** [emporia-vue-local/esphome#50] [Q&A]
Hi,
I'm interested to buy this product since it's possible to have all the data locally with this "custom firmware".
But my house is receiving the current via 3 wires only (3x230v) without neutral, it's called "delta" systems.
Emporia is telling that this kind of systems is not handled by the official firmware/app.

So I was wondering if this "custom firmware" (esphome + custom component + configuration file tweaked for my case) could handle it?
If it does, what could be the impact on the configuration file?
And if it doesn't, as I already have a global consumption measure with my solar inverter systems, could I use this "custom firmware" to have detailed analysis of 19 circuits (16 small clamps + 3 big clamps)?

Thanks in advance.

**Comment by flaviut** [comment]
I don't think that'll work. At the moment, the ESPHome software for the Vue only reliably measures power, which requires also measuring voltage. Voltage is measured between each phase and neutral.

There's [someone who's trying to get direct current data](https://github.com/emporia-vue-local/esphome/discussions/49), which would let you measure the current passing through each clamp. So far, their results are off by about 10% for unknown reasons.

I would talk to an licensed electrician first, this is probably insane and dangerous, but perhaps if you don't care about one of the phases, you can wire this system up as a 2-phase system with one of the phases being treated as neutral?

**Reply by MarkoPaasila** [reply]
would it give approximately right values by using a different calibration or multiply -filter?

**Reply by flaviut** [reply]
I don't know--I don't understand how these systems work.

The way the vue works is it measures current and voltage many times per second using [Riemann sums](https://en.wikipedia.org/wiki/Riemann_sum), and multiplies the sums when it reports a new datapoint.

I don't know where or how you would measure a voltage inside a delta system, since there is no common. Perhaps there is some way for you to accomplish this with a series of 1:1 transformers, but I am not an electrical engineer & mains voltage is scary.

**Reply by MarkoPaasila** [reply]
Right. I have the opposite situation: voltages are measured against N and I have 3~ motors in Delta configuration. I get approximately the correct power simply by multiplying eith sqrt(3).

**Comment by MarkoPaasila** [comment]
Is it legal and possible to create a floating zero and/or connect the N wire of emporia to  that or earth? That would give you the voltages emporia needs.

**Comment by flaviut** [comment]
Apparently the Vue will boot up without a neutral connection at all: https://github.com/emporia-vue-local/esphome/discussions/197#discussioncomment-6773530

I'm not certain that that helps here, but I found it surprising!
